src/providers/WorkflowCore.Providers.Redis/WorkflowCore.Providers.Redis.csproj

```diff
@@ -2,18 +2,18 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Version>3.0.2</Version>
+    <Version>3.0.3</Version>
     <PackageLicenseUrl>https://github.com/danielgerlag/workflow-core/blob/master/LICENSE.md</PackageLicenseUrl>
     <RepositoryUrl>https://github.com/danielgerlag/workflow-core.git</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <PackageProjectUrl>https://github.com/danielgerlag/workflow-core</PackageProjectUrl>
     <Description>Redis providers for Workflow Core (Persistence, queueing, distributed locking and event hubs)
 </Description>
-    <PackageVersion>3.0.2</PackageVersion>
+    <PackageVersion>3.0.3</PackageVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.2.0" />
```

Comment:
Do we need to update the redis package to support the EF upgrade?

Reply:
It might be that it runs ok with mixed versions, but EF Core 5.0 requires Microsoft.Extensions.Logging => 5.0.0 which requires the abstractions package => 5.0.0.

I don't think this is a difficult upgrade for clients to use 5.0.0 instead of 2.2.0, and it seems better than leaving the redis package on a different version to the EF Core related libs?

Reply:
if the consumer of the redis package does not use the EF packages, the minimum version of the Abstraction packages could be 2.2.0, but if they add the EF packages that require 5.0, they'd be forced to update their own app to use 5.0, which should still be compatible with 2.2.0 of the abstractions that the redis package depends on.